R/module_filter_manager.R

```diff
@@ -164,9 +164,10 @@ filter_manager_srv <- function(id, filtered_data_list, filter) {
           rownames(mm) <- ""
         }
 
-        mm
+        # Report Previewer will not be displayed.
+        mm[!grepl("Report previewer", names(mm))]
```

Comment:
> Not always because one can run an app with modules that do not use reporters and name the first module "Report previewer", however unlikely that may be.

I am aware of that. To make it clear, I was thinking something like this:

```suggestion
        # Report Previewer will not be displayed.
        if ("Report previewer" %in% names(mm)[length(mm)]) {
          mm <- mm[, -length(mm)]
        }
        
        mm
```

This way, even though we have a user module with "Report previewer" as the label name, it will not be removed. 
We only check the last element of mm due to current teal behavior.

WDYT?

Reply:
But if you label a module `"Report previewer"` in an app without reporters, it will be removed, while it should not.